client/payment-methods/index.js

```diff
@@ -109,19 +89,13 @@ const PaymentMethods = ( { enabledMethodIds, onEnabledMethodIdsChange } ) => {
 				/>
 			) }
 			<Card className="payment-methods">
-				<CardBody className="payment-methods__enabled-methods-container">
```

Comment:
Before there were some styles applied to `payment-methods__enabled-methods-container`, but by passing `size={ null }` we get the same result.